resources/views/vendor/mail/html/message.blade.php

```diff
@@ -0,0 +1,27 @@
+@component('mail::layout')
+    {{-- Header --}}
+    @slot('header')
+        @component('mail::header', ['url' => config('app.url')])
+            <div class="logo"><img src="{{ asset('images/laravelio.png') }}" title="Laravel.io"></div>
```

Comment:
Perhaps we can use an inline attachment instead? https://laravel.com/docs/5.7/mail#inline-attachments